application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksCommand.java

```diff
@@ -0,0 +1,276 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.component.ButtonInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.BookmarksRecord;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BookmarksCommand extends SlashCommandAdapter {
+
+    public static final long EXPIRE_DELAY_SECONDS = 3L * 24 * 60 * 60; // 3 Days
+
+
+    private static final MessageEmbed EMBED_NOT_IN_DM =
+            new EmbedBuilder().setTitle("This command cannot be used in a DM!")
+                .setColor(0xFF8484)
+                .build();
+
+    private static final MessageEmbed EMBED_NOT_IN_HELP_THREAD =
+            new EmbedBuilder().setTitle("This command can only be used in help threads!")
+                .setColor(0xFF8484)
+                .build();
+
+    private static final MessageEmbed EMBED_ALREADY_BOOKMARKED =
+            new EmbedBuilder().setTitle("You have already bookmarked this thread!")
+                .setColor(0xFF8484)
+                .build();
+
+    private static final MessageEmbed EMBED_BOOKMARK_REMOVED =
+            new EmbedBuilder().setTitle("The bookmark was removed!").setColor(0xFFA500).build();
+
+
+    private final Database database;
+    private static final Map<String, BookmarksPaginator> bookmarksPaginators = new HashMap<>();
+
+
+    /**
+     * Creates a new instance and registers the commands with options
+     */
+    public BookmarksCommand(Database database) {
+        super("bookmarks", "Add or view help thread bookmarks", CommandVisibility.GLOBAL);
+        this.database = database;
+
+
+        SubcommandData addSubCommand = new SubcommandData("add", "Adds a new boomark.")
+            .addOption(OptionType.STRING, "note", "Add a note to your bookmark.", false);
+
+        SubcommandData viewSubCommand = new SubcommandData("view", "Displays your bookmarks.");
+
+        getData().addSubcommands(addSubCommand, viewSubCommand);
+    }
+
+    /*
+     * Slash command handler section
+     */
```

Comment:
i'd say remove this comment

Reply:
theres more?

yes remove all which parts the code in sections

Reply:
ok

Reply:
im gonna do that after #640 gets merged so i dont have to deal with the warnings anymore

Reply:
I have removed the section comments with 36251ef1d7f209bb9ec87718c7ff534791f3d0ed